### PR TITLE
Add December 2022 newsletter

### PR DESCRIPTION
**Pull request by CollierCZ**
----
:books: Documentation preview :books:: https://writethedocs-www--1861.org.readthedocs.build/en/1861/

### CODE REVIEW

File: docs/blog/newsletter-december-2022.rst [hunk 1]
```diff
@@ -0,0 +1,121 @@
+.. post:: December 07, 2022
+   :tags: newsletter
+
+#########################################
+Write the Docs Newsletter – December 2022
+#########################################
+
+Hello to all the lovely documentarians out there. The end of the year is approaching, which is a wonderful time to reflect on all the things we've done in 2022. And on what to look forward to for next year. I hope your personal list of accomplishments is long and your list of remaining things to do isn't quite as daunting as mine.
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2022.rst]: what do you think of addressing the reader directly here? 

> Hello lovely documentarians.

maybe that is _too_ familiar?

Comment (plaindocs) [reply docs/blog/newsletter-december-2022.rst]: And I'd opt for 

> And to reflect on what to look forward to for next year.

or similar

File: docs/blog/newsletter-december-2022.rst [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+This year, Write the Docs has already organized two conferences and is in the process of a third, the `Write the Docs Australia conference </conf/australia/2022/>`__. The new ideas may already be flowing there by the time you read this. And there's still time to contribute to the `2022 salary survey <https://salary-survey.writethedocs.org/>`__. The results from the survey are always helpful in understanding our industry and each of our places in it.
+
+Looking forward at next year, a `venue has been chosen and more details settled for the Portland conference <https://www.writethedocs.org/conf/portland/2023/news/announcing-cfp/>`__. The conference will once again be in person. If you have ideas you'd like to share, the `Call for Proposals <https://www.writethedocs.org/conf/portland/2023/cfp/>`__ is open until the beginning of February. The call is open to documentarians of all stripes, so don't be afraid to send in your proposal.
+
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2022.rst:13]: Let's add a line here that the Atlantic CFP will up sometime Q1 2023 ish? To forestall more questions

File: docs/blog/newsletter-december-2022.rst [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+Another technique is to present the contributor with the problem, not the solution. Rather than suggesting a better sentence, you can ask if they can think of any ways to make the sentence shorter or explain the reasons you find it unclear. This helps them practice self-editing and gives them more ownership over the final piece.
+
+Finally, the work isn’t all on your side: the reviewee needs to be open to feedback too. Consider: Do you officially manage/coach/mentor this person or are you a peer? Are they happy to spend a lot of time iterating or do they just want to get this piece published ASAP? You may have a lot to teach, but make sure that your reviewee is ready to learn.
+
+--------------------------
+When to create screenshots
+--------------------------
+
+Best practices for creating and managing screenshots aren’t always the clearest. Moreover, because of the overhead of maintaining screenshots and the potential accessibility concerns they present, many people avoid creating them despite the benefit they can provide. Still, even with the shared aversion, sometimes screenshots are helpful or even necessary.
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2022.rst]: I'd reword `moreover` as slightly archaic, but that might be a personal opinion

File: docs/blog/newsletter-december-2022.rst [hunk 4]
```diff
@@ -0,0 +1,121 @@
...
+
+When determining what to capture in a screenshot, be selective to keep maintenance low. For example, avoid capturing the entire screen when only a small region is enough to help orient users. You might also focus more on screenshotting a conceptual flow. Even consider the screenshot’s value. Is the text already clear or will the screenshot make a procedure more understandable?
```
Comment (plaindocs) [inline docs/blog/newsletter-december-2022.rst]: are we sure we want "screenshotting"?

**Review by plaindocs — APPROVED** [review]
Looks good, I have minor nits only.

File: docs/blog/newsletter-december-2022.rst [hunk 5]
```diff
@@ -0,0 +1,121 @@
...
+
+Users of the pull method said that it's difficult to keep up with everything you have to monitor, from email and Slack discussions to design documents and code updates. At the same time, you don't have to wait for others to tell you what needs documenting or rely on their judgment about what information is relevant. Attending team meetings, watching project tracking tools, and reviewing product roadmaps are especially helpful when you follow the pull method.
+
+On the other side of the coin, the push method narrows the range of resources you need to monitor but requires trust and good relationships with those who know the most about the product. Processes for the push method can include regular meetings about documenting planned features with product managers or owners, patterns for getting notified about known docs needs and anticipated updates, and even having documentation included in the definition of done for product releases.
```
Comment (hillaryfraley) [inline docs/blog/newsletter-december-2022.rst]:
```suggestion
On the other side of the coin, the push method narrows the range of resources you need to monitor but requires trust and good relationships with those who know the most about the product. Processes for the push method can include regular meetings about documenting planned features with product managers or owners, patterns for getting notified about known docs needs and anticipated updates, and even including documentation in the definition of done for product releases.
```
